api/repositories/broker_parameters_client.go

```diff
@@ -0,0 +1,55 @@
+package repositories
+
+import (
+	"context"
+	"fmt"
+
+	korifiv1alpha1 "code.cloudfoundry.org/korifi/controllers/api/v1alpha1"
+	"code.cloudfoundry.org/korifi/controllers/controllers/services/osbapi"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+type ParamsResult struct {
+	Parameters map[string]any
+}
+
+type BrokerParamsClient struct {
+	clientFactory osbapi.BrokerClientFactory
+	k8sClient     client.Client
+	rootNamespace string
+}
+
+func NewBrokerParamsClient(clientFactory osbapi.BrokerClientFactory, k8sClient client.Client, rootNamespace string) *BrokerParamsClient {
+	return &BrokerParamsClient{
+		clientFactory: clientFactory,
+		k8sClient:     k8sClient,
+		rootNamespace: rootNamespace,
+	}
+}
+
+func (c *BrokerParamsClient) GetServiceBindingParameters(ctx context.Context, serviceBinding *korifiv1alpha1.CFServiceBinding) (ParamsResult, error) {
```

Comment:
Could the method simply return `map[string]any, error`? I.e., do we need to wrap the map into `ParamsResult`?